src/components/Navigation/Navigation.scss

```diff
@@ -17,7 +18,7 @@ $nav-max-width: 360px;
   min-height: 100%;
   background-color: var(--p-background);
   -webkit-overflow-scrolling: touch;
-  border-right: var(--p-border-divider);
+  border-right: 1px solid var(--p-divider);
```

Comment:
The `--p-border-divider` var was fixed in #5121 so I think we could leave this as-is now 👍